.github/workflows/unit-tests.yml

```diff
@@ -36,13 +36,25 @@ jobs:
           - "common-with-cms"
           - "xmodule-with-lms"
           - "xmodule-with-cms"
+        mongo-version:
+          - "4.4.13"
+          - "7.0.5"
```

Comment:
Can we pin the minor version, i.e., `4.4` or `7.0`?

Reply:
done